pkg/operator/ceph/test/containers.go

```diff
@@ -132,6 +132,8 @@ func (ct *ContainersTester) AssertEnvVarsContainCephRequirements() {
 					"POD_CPU_REQUEST env var does not have the appropriate source:", e)
 			}
 		}
+		vars := FindDuplicateEnvVars(c)
+		assert.Equal(ct.t, 0, len(vars))
```

Comment:
Please add an additional message to this assert to aid in debugging.

Suggest something like: "found duplicate env vars: %v", vars